### PR TITLE
Add a link to the rerun command.

### PR DESCRIPTION
**Pull request by spxtr**
Took the basic idea from [here](https://www.w3schools.com/howto/howto_css_modals.asp).
ss

If you scroll down and click a rerun button it will scroll you to the top. If you click on one rerun button then all will go purple. Oh well.

#2495

### CODE REVIEW

**Comment by rmmh** [comment]
I'm not super excited about adding a button that's useless to most devs. Can we hide it behind something like a checkbox?

Does deck have more metadata that it could show? We could expand the row to show additional information, like the trigger context/user, execution timeline (time spent queued), etc.

**Comment by spxtr** [comment]
> I'm not super excited about adding a button that's useless to most devs.

https://prow.k8s.io/ is useless to most devs by design. For a more useful interface, see gubernator :)

> Can we hide it behind something like a checkbox?

I don't understand how a checkbox expresses "rerun this job" better than a ⟳.

> Does deck have more metadata that it could show? We could expand the row to show additional information, like the trigger context/user, execution timeline (time spent queued), etc.

Yessss, we should do that. I was looking into angularifying deck to make that neater, since I don't like javascript enough to do it manually. It's not going to happen for this PR, though.

File: prow/cmd/deck/static/script.js [hunk 1]
```diff
@@ -183,6 +191,20 @@ function createLinkCell(text, url) {
     return c;
 }
 
+function createRerunCell(modal, rerun_command, prowjob) {
+    url = "https://" + window.location.hostname + "/rerun?prowjob=" + prowjob;
```
Comment (rmmh) [inline prow/cmd/deck/static/script.js]: var

Comment (spxtr) [inline prow/cmd/deck/static/script.js]: Done, and added `"use strict";` at the top.

**Pull request merged**